Lib/ensurepip/__init__.py

```diff
@@ -12,13 +12,15 @@
 
 _WHEEL_DIR = "/usr/share/python-wheels/"
 
+_wheels = {}
 
 def _get_most_recent_wheel_version(pkg):
     prefix = os.path.join(_WHEEL_DIR, "{}-".format(pkg))
-    suffix = "-py2.py3-none-any.whl"
-    pattern = "{}*{}".format(prefix, suffix)
-    versions = (p[len(prefix):-len(suffix)] for p in glob.glob(pattern))
-    return str(max(versions, key=distutils.version.LooseVersion))
+    _wheels[pkg] = {}
+    for suffix in "-py2.py3-none-any.whl", "-py3-none-any.whl":
+        pattern = "{}*{}".format(prefix, suffix)
+        _wheels[pkg].update({p[len(prefix):-len(suffix)]: os.path.basename(p) for p in glob.glob(pattern)})
```

Comment:
That might be too clever.
```suggestion
        for p in glob.glob(pattern):
            version_str = p[len(prefix):-len(suffix)]
            _wheels[pkg][version_str] = os.path.basename(p)
```

Reply:
Thanks, you are most likely correct. I was adapting and adapting and adapting and I should have taken a step back. Let's also call it `path`  instead of `p`?

Reply:
Yup, wizard mode is not easy to exit :)

Yes, `path` is better.

Reply:
![not important stolen image of a snake wizard](https://i.pinimg.com/236x/c1/ad/0c/c1ad0c9d5ef8e06e0d117b08ba6e7254.jpg)